src/.jshintrc

```diff
@@ -6,21 +6,21 @@
 	"expr": true,
 	"immed": true,
 	"noarg": true,
-	"onevar": true,
 	"quotmark": "double",
-	"smarttabs": true,
-	"trailing": true,
```

Comment:
These options are no longer supported by JSHint. JSCS should be used to check such things.